src/vue/pages/Issuance.vue

```diff
@@ -84,9 +84,9 @@
     </div>
     <div class="issuance-history__collection-loader">
       <collection-loader
-        :first-page-loader="loadHistoryWrapper"
-        @first-page-load="loadHistory"
-        @next-page-load="concatHistory"
+        :first-page-loader="getIssuanceOperationsResponse"
```

Comment:
here `getHistory` would be enough. `setHistory` and `extendHistory` would be good either

Reply:
Resolved, renamed